src/frontends/paddle/src/op/one_hot_v2.cpp

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "default_opset.hpp"
+#include "openvino/frontend/paddle/node_context.hpp"
+
+namespace ov {
+namespace frontend {
+namespace paddle {
+namespace op {
+NamedOutputs one_hot_v2(const NodeContext& node) {
+    auto data = node.get_input("X");
+    Output<Node> depth;
+    if (node.has_input("depth_tensor")) {
+        auto depth_value = node.get_input("depth_tensor");
+        depth = std::make_shared<default_opset::Squeeze>(depth_value);
+    } else {
+        const auto depth_value = node.get_attribute<int>("depth", -1);
+        if (depth_value == -1) {
```

Comment:
Why should we consider the ```depth``` as -1 or a negative, is it a available value in Paddle‘s model ? if yes, could you help to add a new case in uni-test ?

Reply:
Like I mentioned before, input without `num_class` is not permitted in Paddle‘s model and negative value is also not available. So I think it is better not to set a default value for `num_class`. If it must has a default value, I would recommend `-1`, same as the [pytorch implement](https://pytorch.org/docs/stable/generated/torch.nn.functional.one_hot.html?highlight=one_hot#torch.nn.functional.one_hot).
> num_classes ([int](https://docs.python.org/3/library/functions.html#int)) – Total number of classes. If set to -1, the number of classes will be inferred as one greater than the largest class value in the input tensor.

Reply:
Thanks. Could you help to confirm if paddle will throw an exception when we set the ```num_class``` as -1/negative ? If Paddle will reject this case, i think we can remove the default value in OV's frontend.

Reply:
It is not valid when set the `num_class` as -1/negative
![image](https://user-images.githubusercontent.com/69072522/225845341-0226d0c5-8d9b-4cec-8b15-1649753184e5.png)
I have removed default value as well.